dperson/torproxy
# SEC-527 Update Torproxy image base image and packages

## Description of work done
This PR makes updates with the intent of rebuilding the published image.

The packages updated result in some config changes because the new privoxy config files end in `.new`, this PR accounts for that.

## Testing notes
- Pull the branch and build the image.
- Exec the image e.g. `docker run -it -p 8118:8118 -p 9050:9050 -d torproxytest`
- Configure your local proxy to use `localhost:8118` and confirm you can still browse through the local Tor relay.
- Exec into the container and confirm the following processes are running:
```sh
PID   USER     TIME  COMMAND
    1 root      0:00 /sbin/tini -- /usr/bin/torproxy.sh
    7 tor       0:06 /usr/bin/tor
   17 privoxy   0:00 /usr/sbin/privoxy --user privoxy /etc/privoxy/config
  350 root      0:00 bash
  586 root      0:00 /bin/sh -c curl -sx localhost:8118 'https://check.torproject.org/' |             grep -qm1 Congratulations
  592 root      0:00 curl -sx localhost:8118 https://check.torproject.org/
  593 root      0:00 grep -qm1 Congratulations
  595 root      0:00 ps
```
